testing/features/callout.feature

```diff
@@ -7,30 +7,50 @@ Feature: Callout component
     Background:
       Given a Standard Callout component is on the page
 
-    Scenario: Callout contains a title and message
+    Scenario: English Callout contains a title and message
```

Comment:
The translated parts of the Callout are the labels (currently in CSS and due for extraction), the rest of the content is user provided, so similarly to the OISC warnings you might find that some of these tests aren't needed.

Reply:
This test just validates presence of title / message. The label is the one interrogated. So these should be fine.